### PR TITLE
Adds wounds to the medical huds

### PR DESCRIPTION
**Pull request by NithaIsTired**
## About The Pull Request
Adds wounds to the medical huds.
Sprites by [Tamamitsune](https://github.com/Tamamitsune)

## Why It's Good For The Game
Having a way to tell that people are wounded will surely decrese the chances of medical sending people off after just doing basic healing, also making it very easy to prioritize patients.

## Changelog

:cl: NithaIsTired, Tamamitsune
add: Medical huds now displays the current wound severity the patient has.
/:cl:

### CODE REVIEW

**Comment by NithaIsTired** [comment]
> would be best upstream on tg i think theyd like this

I might do that once I'm not too sick to do stuff

**Comment by Wolf-751** [comment]
This is a damn good idea as a medbay main i can tell you though the best wound you want to cover for this is bleeding,bullet holes,bones are alright but not that dire to treat instantly and usally a quick look over will tell you there broken bone area the most pressing wounds are the bleeding/rotting ones.

**Comment by nikothedude** [comment]
also it might be best to instead give the wound datum a icon variable and a icon severity for better OOP practices and also to solve the issue above, so that weeping avulsions will override a compound fracture

alternatively you can have the icons cycle through the highest severity ones

**Comment by NithaIsTired** [comment]
I did consider adding specific bleeding icons but I was already suffering to make this cycle with the virus icons so i decided to keep it simple for now (bleeding is easy to spot by just looking at the person anyways but the wound icon will still show)

I'll see if i can get an icon for bleeding and try to make that work too

File: modular_zubbers/code/game/data_huds.dm [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		for (var/datum/wound/wound in all_wounds)
+			if (wound.severity == WOUND_SEVERITY_CRITICAL)
+				holder.icon_state = "hudcriticalwound"
+				highest_severity_found = TRUE
+			else if (wound.severity == WOUND_SEVERITY_SEVERE && !highest_severity_found)
+				holder.icon_state = "hudseverewound"
+				highest_severity_found = TRUE
+			else if (wound.severity == WOUND_SEVERITY_MODERATE && !highest_severity_found)
+				holder.icon_state = "hudmoderatewound"
+				highest_severity_found = TRUE
```
Comment (Majkl-J) [inline modular_zubbers/code/game/data_huds.dm:54]:
```suggestion
			if (wound.severity == WOUND_SEVERITY_CRITICAL)
				holder.icon_state = "hudcriticalwound"
				highest_severity_found = TRUE
				break
			else if (wound.severity == WOUND_SEVERITY_SEVERE && !highest_severity_found)
				holder.icon_state = "hudseverewound"
				highest_severity_found = TRUE
			else if (wound.severity == WOUND_SEVERITY_MODERATE && !highest_severity_found)
				holder.icon_state = "hudmoderatewound"
				highest_severity_found = TRUE
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Review one, very brief look. There will probably be many more

**Comment by nevimer** [comment]
This is good, we will be testing it for a bit. Try to get around to reviews when you can.

**Comment by Tupinambis** [comment]
Should have been a thing when wounds was originally added. Good change.

**Comment by Swiftfeather** [comment]
Hey there, this has been stale for quite a while. If you decide to continue working on this please feel free to re-open this PR!

**Pull request closed without merging**